src/components/UploadDropzone.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from 'react'
+import { useEffect } from 'react'
+import { useDropzone } from 'react-dropzone'
+import styled from 'styled-components'
+
+import { Button, ButtonType } from './Button'
+
+type FileRejections = {
+  file: File
+  errors: any
+}
+
+type UploadDropzoneProps = {
+  onCancel: () => void
+  onSubmit: () => void
+  setData: (items: any) => void
+  acceptedFilesToUpload?: string
+  maxFilesToUpload?: number
+}
+
+// const PDF = 'application/pdf'
+const IMAGE = 'image/*'
+
+export const UploadDropzone = ({
+  onCancel,
+  onSubmit,
+  setData,
+  acceptedFilesToUpload,
```

Comment:
I would rename that to `acceptedFileTypes` as type Array<string>. That would be more clear to me how to use the component.